langchain4j-core/src/main/java/dev/langchain4j/audit/api/listener/LLMInteractionCompleteEventListener.java

```diff
@@ -0,0 +1,17 @@
+package dev.langchain4j.audit.api.listener;
+
+import dev.langchain4j.audit.api.event.LLMInteractionCompleteEvent;
+
+/**
+ * A listener for {@link LLMInteractionCompleteEvent}, which represents an event
+ * that occurs upon the completion of an interaction with a large language model (LLM).
+ * This interface extends the generic {@link LLMInteractionEventListener} interface,
+ * allowing it to specifically listen for completion events.
+ */
+@FunctionalInterface
+public interface LLMInteractionCompleteEventListener extends LLMInteractionEventListener<LLMInteractionCompleteEvent> {
```

Comment:
```suggestion
public interface LLMInteractionCompletedEventListener extends LLMInteractionEventListener<LLMInteractionCompletedEvent> {
```
Complete -> Completed

Reply:
Done